build/run_unit_group.sh

```diff
@@ -64,7 +64,7 @@ alias echo='{ [[ $- =~ .*x.* ]] && trace_enabled=1 || trace_enabled=0; set +x; }
 
 # Test Groups  -- start --
 function test_group_broker_group_1() {
-  mvn_test -pl pulsar-broker -Dgroups='broker' -DtestReuseFork=true -DskipAfterFailureCount=1
+  mvn_test -pl pulsar-broker -Dgroups='broker' -DtestReuseFork=true -DskipAfterFailureCount=1 -DforkCount=0
```

Comment:
this is not related to the pull. please revert

Reply:
yes, right. We find the test failed not related to zk(may be fixed by this patch), but not reproducible locally.  So we try to modify some parameters that may affect them.